Sources/NetworkProtection/Status/ConnectionErrorObserver/ConnectionErrorObserverThroughSession.swift

```diff
@@ -97,9 +97,9 @@ public class ConnectionErrorObserverThroughSession: ConnectionErrorObserver {
     // MARK: - Obtaining the NetP VPN status
 
     private func updateTunnelErrorMessage(session: NETunnelProviderSession) throws {
-        try session.sendProviderMessage(.getLastErrorMessage) { [weak self] (errorMessage: ExtensionMessageString?) in
+        /*try session.sendProviderMessage(.getLastErrorMessage) { [weak self] (errorMessage: ExtensionMessageString?) in
```

Comment:
Oh I should enable this back I think?  But strangely enough this has eliminated all the crashing I was seeing.

Reply:
This is an interesting one because I've been having problems with it on iOS today as well. Let's sync about it on Tuesday after Apple Weekly.

Reply:
Re-enabled.